CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - Added new species ALK4N1 and ALK4N2 to Ox family in KPP
 - Added Cloud-J input parameters to geoschem_config.yml in new photolysis sub-menu called cloud-j
 - Added computation of water concentration to use in photolysis for application of UV absorption by water in Cloud-J v8
+- Added `Is_Photolysis: true` tags to APINP, AROMCHO, BPINOOH, BPINP, C96O2H, H2O, HO2, LIMAL, LIMO3H, PINAL, PINONIC, SO2 in `species_database.yml`
```

Comment:
I think we need a line saying these species (and others?) were added to species database following Travis et al. Then this line can be removed since the modification to add `Is_Photolysis` was included in the same version before release.